zstacklib/zstacklib/network/ovs/controller.py

```
@@ -0,0 +1,502 @@
+    except OvsError as err:
+        logger.error(f'Get Ovs controller failed. {err}')
+        raise
+
+
+def is_vm_use_openvswitch(vm_uuid: str) -> bool:
+    """Check if a VM is using Open vSwitch."""
+    try:
+        vm_interface_list = shell.call(f'virsh domiflist {vm_uuid}').strip()
+        if 'vhostuser' in vm_interface_list:
+            return True
+        return False
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)